### PR TITLE
Upgraded to PureScript 0.14.3

### PR DESCRIPTION
**Pull request by icyrockcom**
Upgrade to PureScript 0.14.3. Split out of an old PR https://github.com/exercism/purescript/pull/212.

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
@icyrockcom I've requested a review from @aimorris, who is the maintainer of the purescript track. Should there not be a review within 5 days or so, ping me and we can get this merged.

**Comment by icyrockcom** [comment]
Thanks @ErikSchierboom. @aimorris Just wanted to check if you had a chance to review. It'd be great to get a review before the merge, but it also would be good to not have this fall into oblivion again. :)

**Review by ErikSchierboom — APPROVED** [review]
Let's go!

**Pull request merged**
